servicetalk-http-router-predicate/src/test/java/io/servicetalk/http/router/predicate/PredicateRouterOffloadingTest.java

```diff
@@ -132,7 +133,7 @@ void predicateOffloadedAndNotRoute(RouteServiceType routeServiceType) throws Exc
         this.routeServiceType = routeServiceType;
         assumeSafeToDisableOffloading(routeServiceType);
         final HttpPredicateRouterBuilder routerBuilder = newRouterBuilder();
-        serverBuilder.executor(executionContextRule.executor());
+        serverBuilder.executionStrategy(offloadAll()).executor(executionContextRule.executor());
```

Comment:
Hm, after noticing this change I'm not sure if we are ready to override `requireOffloads()` now because it changes the default behavior.
By returning `offloadNone()`, the default behavior will result in no offloading for all routes if the strategy is not specified at the builder. This is not what we need for the default behavior.
If a custom strategy is configured at the builder (like `offloadNone()`), it will take priority.

Reply:
> By returning offloadNone(), the default behavior will result in no offloading for all routes if the strategy is not specified at the builder. This is not what we need for the default behavior.

I am fine with leaving the behaviour as it currently is. A note on the class to specify `offloadNone()` to change the default behaviour may be sufficient.

Reply:
The other option would be for the service to require no offloads but the default "unspecified" behavior for routes would be to offload. This would achieve the same effect but not require use of the builder `offloadNone()`

Reply:
After discussion we don't want to change the behavior. The suggested behavior of having the router require no offloads and offloading happen by default for routes that do not specify offloading would be better but it is not appropriate to change the existing behavior.